.gitlab-ci.yml

```diff
@@ -35,8 +35,27 @@ default:
         - lstopo --of xml >$(hostname).xml
         - export FLUX_HWLOC_XMLFILE=$(pwd)/$(hostname).xml
         - !reference ['.build-core', 'script']
-        - cd $FLUX_BUILD_DIR
-        - make -j 32 check
+        - cd $CORE_BUILD_DIR
+        - make -j check
+
+.test-pmix:
+    extends: .lc-variables
+    variables:
+        PYTHON: "/usr/bin/python3"
+        debug: t
+        FLUX_TESTS_LOGFILE: t
+    script:
+        - !reference ['.build-core', 'script']
+        - cd $CORE_BUILD_DIR
+        - export PKG_CONFIG_PATH=$(pwd)/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
+        - module load openmpi
+        - export PKG_CONFIG_PATH=/usr/tce/packages/openmpi/openmpi-4.1.2-intel-classic-2021.6.0-magic/lib/pkgconfig:$PKG_CONFIG_PATH 
```

Comment:
Since `mpicc` is not a directory, you'd have to take the dirname first:
```console
$ which mpicc
/usr/tce/packages/openmpi/openmpi-4.1.2-intel-classic-2021.6.0-magic/bin/mpicc
$ echo $(dirname $(which mpicc))
/usr/tce/packages/openmpi/openmpi-4.1.2-intel-classic-2021.6.0-magic/bin
$ ls $(dirname $(which mpicc))/../lib/pkgconfig
ompi-c.pc    ompi-f77.pc  ompi-fort.pc  orte.pc
ompi-cxx.pc  ompi-f90.pc  ompi.pc       pmix.pc
```